cmd/frontend/internal/httpapi/api_test.go

```diff
@@ -26,7 +29,17 @@ func newTest(t *testing.T) *httptestutil.Client {
 	rateLimitStore, _ := memstore.New(1024)
 	rateLimiter := graphqlbackend.NewRateLimiteWatcher(logger, rateLimitStore)
 
-	return httptestutil.NewTest(NewHandler(database.NewMockDB(),
+	db := database.NewMockDB()
+	gh := webhooks.GitHubWebhook{
+		DB: db,
+	}
```

Comment:
```suggestion
	gh := webhooks.GitHubWebhook{DB: db}
```

Disk space is cheap, but nothing's free